git_machete/git_operations.py

```diff
@@ -312,7 +312,8 @@ def __ensure_config_loaded(self) -> None:
 
     def get_config_attr_or_none(self, key: str) -> Optional[str]:
         self.__ensure_config_loaded()
-        return self.__config_cached.get(key.lower())
+        value = self.__config_cached.get(key.lower())
+        return value.strip() if value else value
```

Comment:
Hmmm I wouldn't indiscriminately `strip` all retrieved config values... I can't now think of any that include leading/trailing whitespace on purpose, but still, that might be error-prone

Reply:
I'd rather `strip` case-by-case, as in `get_url_of_remote`